recipe/build_libtorch.sh

```diff
@@ -97,6 +98,11 @@ fi
 if [[ "$PKG_NAME" == "pytorch" ]]; then
   PIP_ACTION=install
   sed "s/3.12/$PY_VER/g" build/CMakeCache.txt.orig > build/CMakeCache.txt
+  if [[ "$PY_VER" == "3.8" ]]; then
```

Comment:
```suggestion
  # h-vetinari - May 2024 
  # We use a fan-out build to avoid the long rebuild of libtorch
  # However, the location of the numpy headers changes between python 3.8
  # and 3.9+ since numpy 2.0 only exists for 3.9+
  if [[ "$PY_VER" == "3.8" ]]; then
```